tests/func/test_api.py

```diff
@@ -228,3 +229,128 @@ def test_open_from_remote(tmp_dir, erepo_dir, cloud, local_cloud):
         remote="other",
     ) as fd:
         assert fd.read() == "foo content"
+
+
+@pytest.fixture
```

Comment:
Let's also separate the tests, since the code is now. E.g. `tests/func/api/test_params.py`

Reply:
Done